.travis.yml

```diff
@@ -28,10 +30,8 @@ env:
   - CXX=g++-4.8 DB="maria mysql mysql2 postgres sqlite3 oracledb" KNEX_TEST_TIMEOUT=60000 ORACLE_HOME=/u01/app/oracle/product/11.2.0/xe ORACLE_SID=XE OCI_LIB_DIR=/u01/app/oracle/product/11.2.0/xe/lib LD_LIBRARY_PATH=/u01/app/oracle/product/11.2.0/xe/lib
 
 addons:
-  postgresql: '9.4'
+  postgresql: '9.6'
```

Comment:
Is this needed for the feature? If so, it's a BC break and it should be a major version bump, and documented.